perllib/FixMyStreet/Cobrand/Hackney.pm

```diff
@@ -282,6 +282,16 @@ sub open311_extra_data_include {
           value => $row->category },
     ];
 
+    if ($contact->email eq "Environment-Graffiti") {
+        push @$open311_only,
+            { name => 'graffiti_offensive', value => $row->get_extra_field_value('offensive') || '' };
```

Comment:
Could the extra field values match the names needed by open311 adapter to save this being needed? (Or the names in open311-adapter changed to match the FMS values?) Or am I missing something?

Reply:
Yeah I'm not sure what my thinking was, in hindsight. Think I wanted to preserve the field names from live for some reason but make it obvious what they were for in open311-adapter. Anyway have reverted this and used the `graffiti_`-prefixed field names in FMS.